types/semantic-release/index.d.ts

```diff
@@ -422,8 +425,23 @@ declare namespace SemanticRelease {
          * The shared logger instance of semantic release.
          */
         logger: {
-            log: (message: string, ...vars: any[]) => void,
-            error: (message: string, ...vars: any[]) => void,
+            await: LoggerFunction;
```

Comment:
where all those came from? it looks logger api is little simpler:
https://github.com/semantic-release/semantic-release/blob/58a226f29c04ee56bbb02cc661f020d568849cad/test/index.test.js#L36-L40

Reply:
semantic release uses signale to do the logging as you can see [here](https://github.com/semantic-release/semantic-release/blob/master/lib/get-logger.js). Signale provides their own typings, so I just took it from [their repo](https://github.com/klaussinani/signale/blob/master/types/signale.d.ts#L105)

You can see below all methods added on the MR when I console.log the logger object
![image](https://user-images.githubusercontent.com/7675500/178171438-7c5849e7-074e-4095-a96e-47be61028e89.png)

The simpler portion you saw is just what they decided to test I imagine (and probably only what they use internally as well)

Reply:
ok, sorry. They do not have much docs (which should indicate what should be used)